### PR TITLE
Pause doc: Trade quest marker

### PR DESCRIPTION
**Pull request by Dragorn421**
This is this triangle thing going up and down and flashing yellow-red:

<details>

image

image

image

image

</details>

### CODE REVIEW

**Review by fig02 — APPROVED** [review]
I think I would prefer "marker" or "cursor" over just "location", but dont have that strong of an opinion

**Comment by Dragorn421** [comment]
I'm not attached to location, `tradeQuestLocation` is an existing name in main, I can change it

"cursor" is already used in the context of pause (it's the... cursor controlled by the player), so "marker" ?

**Review by krm01 — APPROVED** [review]
TIL this feature even exists

**Comment by cadmic** [comment]
for me marker > location >> cursor

**Review by cadmic — APPROVED** [review]

**Comment by Dragorn421** [comment]
Will change to "marker"

File: include/z64pause.h [hunk 1]
```diff
@@ -184,7 +186,7 @@ typedef struct PauseContext {
     /* 0x0262 */ s16 promptChoice; // save/continue choice: 0 = yes; 4 = no
     /* 0x0264 */ s16 ocarinaSongIdx;
     /* 0x0266 */ u8 worldMapPoints[20]; // indices: `WorldMapPoint` enum, values: `WorldMapPointState` enum
-    /* 0x027A */ u8 tradeQuestLocation;
+    /* 0x027A */ u8 tradeQuestMarker;
```
Comment (cadmic) [inline include/z64pause.h:189]: maybe comment that this takes `WORLD_MAP_POINT_` or is `TRADE_QUEST_MARKER_NONE`?

**Review by cadmic — APPROVED** [review]
Thanks

**Pull request merged**